LICENSE

```diff
@@ -1,19 +1,380 @@
 
-For licensing information of the libdeltachat library
-please see src/LICENSE 
+The files in this directory and under the "src", "python", "doc" and "cmdline" subdirectories
```

Comment:
negative matching ("all but the libs directory") might be easier?  not that i mind either way